contents/blog/measuring-feature-success.md

```diff
@@ -0,0 +1,114 @@
+
+![Sampling UI](../images/blog/measuring-feature-success/sampling.png)
+
+At this stage, feature success is broad usage. A significant portion of target users should be using the feature. What this portion is depends on the usage of similar features. The feature should also be issue free and work as intended.
+
+To evaluate this, we watch session replays and track metrics that match specific criteria we care about. For example, we might look at the session recordings for high ICP, paid users in the 99th percentile of feature usage to see how power users are utilizing the product. We also might create a dashboard to show usage and the feature’s impact (such as average insight load time).
+
+In Yakko’s case, he: 
+
+- created a key metrics dashboard that he would dig into a few times a day.
+- watched session replays of the feature to see if it was having the intended impact or causing any confusion (he fixed a few visual regressions as a result of this).
+- used the [user interview app](/docs/apps/user-interview) to prompt users who tried sampling for a user interview.
+
+![Dashboard](../images/blog/measuring-feature-success/dashboard.png)
+
+After doing all these, he was confident the feature was trending towards success and continued the rolling out until he hit 100% of users.
+
+### 4. Post-launch
+
+Success after launch looks like continued growth in usage metrics. It also looks like recommendations or public praise for that feature. If the feature has a large enough impact, it may even inspire a marketing push or case study. 
+
+If a large feature is successful to warrant dedicated, sustained work, such as session recordings or group analytics, we might plan to [charge for it](/handbook/engineering/feature-pricing). If this is the case, then success becomes users paying for it. As an example, charging for our feature flags product is a goal for Q2 2023. Follow-on goals are likely growing the revenue of the product. 
+
+The success of the feature post-launch determines future effort and investment in that feature or related features. If a feature gets a lot of usage and feedback, then we often develop plans to continue working on ideas related to it.
+
+Sampling is a relatively self-contained feature. It likely won’t get charged for. Yakko continues to ask for feedback about sampling, but more work will go into improving query performance generally as it continues to be a challenge for some. The success of sampling led to more investment in improving query performance.
+
+## How you can measure feature success
+
+One of the largest factors in how to measure feature success is the size of your feature, product, and company.
+
+- For early-stage startups and small features, it means obvious increases in key metrics.
```

Comment:
maybe good to say what those key metrics might be? "it means an obvious increases in key metrics such as feature adoption or revenue"